api/src/migrations/1537801450876-add-project-user-role.ts

```
@@ -1,29 +1,56 @@
+            FOREIGN KEY ("project_id") REFERENCES "project"("id") ON DELETE CASCADE,
+            FOREIGN KEY ("user_id") REFERENCES "user"("id") ON DELETE CASCADE
+          )`,
+        );
+
+        await queryRunner.query(`INSERT INTO "project_user" SELECT * FROM "project_user_temp"`);
+        await queryRunner.query(`DROP TABLE "project_user_temp"`);
+
         break;
       default:
-        console.log('Unknown DB type');
+        throw new Error('Unknown DB type: ' + config.db.default.type);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)